assets/src/util/operatorFormatting.tsx

```diff
@@ -0,0 +1,37 @@
+import { joinTruthy } from "../helpers/dom"
+import { Vehicle } from "../realtime"
+
+export const DefaultFallbackString = "Not Available"
```

Comment:
**nitpick:** as a `const` I think this should follow our usual capitalization conventions (`defaultFallbackString`).

Reply:
9c174e5575e4c5061d5be97bed41fc5ea26dfbfe